l10n/en-US/viewer.properties

```diff
@@ -41,6 +41,10 @@ print.title=Print
 print_label=Print
 save.title=Save
 save_label=Save
+# LOCALIZATION NOTE (download_button.title): This string (as a verb) is used in Firefox for Android as a tooltip.
+download_button.title=Download
+# LOCALIZATION NOTE (download_button_label): This string (as a verb) is used in Firefox for Android. Length of the translation matters since we are in a mobile context, with limited screen estate.
```

Comment:
```suggestion
# LOCALIZATION NOTE (download_button.title): used in Firefox for Android as a label for the download button (“download” is a verb). 
# Length of the translation matters since we are in a mobile context, with limited screen estate.
```